.github/actions/constellation_create/action.yml

```diff
@@ -211,8 +211,8 @@ runs:
       shell: bash
       run: |
         constellation init --force --debug
-        echo "KUBECONFIG=$(pwd)/constellation-admin.conf" >> $GITHUB_OUTPUT
-        echo "MASTERSECRET=$(pwd)/constellation-mastersecret.json" >> $GITHUB_OUTPUT
+        echo "KUBECONFIG=$(pwd)/constellation-admin.conf" | tee -a $GITHUB_OUTPUT
```

Comment:
We should not print the Kubernetes config file during the e2e test

Reply:
this only prints the path.